src/tools/cephfs/JournalTool.cc

```diff
@@ -63,7 +63,10 @@ void JournalTool::usage()
     << "General options:\n"
     << "  --rank=filesystem:mds-rank  Journal rank (required if multiple\n"
     << "                              file systems, default is rank 0 on\n"
-    << "                              the only filesystem otherwise.\n"
+    << "                              the only filesystem otherwise.)\n"
+    << "  --type=purge_queue          Journal type (type=purge_queue means\n"
```

Comment:
Thank you for the RP. 
I think using "--purge_queue" is better. we may add "event get --type=XXX list" support later,  two --type options are confusing.

otherwise, the RP looks good

Reply:
Thanks for your quick review, 
can i change --type to --journal_type ?
I've consider two things
1. maybe someday mds need to use "Journaler" again to implement a disk queue.
    if that happend, only need to do little change to the code
2. i prefer to keep this a general option like "--rank"

Reply:
@batrick what's your option?

Reply:
Perhaps `--journal=<"mdlog"|"purge_queue">`?

Reply:
yes! that would be nice

Reply:
it's done